fragments/form/template.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <themecleanflex-components-block v-bind:model="model">
+    <div v-bind:data-per-path="model.path">
+      <div id="formFailure">
+        <p data-per-inline="failureText">{{failureText}}</p>
+      </div>
+      <form v-on:submit="defaultSubmit">
+        <vue-form-generator v-bind:model="formModel" v-bind:schema="schema" v-bind:options="formOptions"></vue-form-generator>
+        <input type="submit">
+      </form>
+    </div>
+  </themecleanflex-components-block>
+</template>
+
+<script>
+export default {
+    props: ['model'],
+    data() {
+      return {
+        formModel: {},
+        formOptions: {
+          validateAfterChange: true
+        },
+        failureText: ''
+      }
+    },
+    methods: {
+      defaultSubmit(e) {
+        e.preventDefault()
+        if(this.model.submitfunction != 'defaultSubmit' && this.model.submitfunction != '') {
+          if(window[this.model.submitfunction]) {
```

Comment:
do we really have to do this as 2 cases?

Reply:
Wrote a better loop to eliminate the extra case